packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -137,6 +138,17 @@ function sfdxProjectConfig(): string {
   return pathToSFDXProjectConfig;
 }
 
+function sfdxGlobalConfig(): string {
+  const pathToSFDXGlobalConfig = path.join(os.homedir(), relativeStateFolder(), SFDX_CONFIG_FILE);
+  return pathToSFDXGlobalConfig;
+}
+
+function sfdxConfig(): string {
+  const pathToSFDXConfig = workspaceUtils.hasRootWorkspace()
+  ? sfdxProjectConfig() : sfdxGlobalConfig();
+  return pathToSFDXConfig;
+}
+
```

Comment:
When I save this file in my IDE, these two blocks are formatted slightly differently.  What do you think about the following formatting:

```suggestion
function sfdxGlobalConfig(): string {
  const pathToSFDXGlobalConfig = path.join(
    os.homedir(),
    relativeStateFolder(),
    SFDX_CONFIG_FILE
  );
  return pathToSFDXGlobalConfig;
}

function sfdxConfig(): string {
  const pathToSFDXConfig = workspaceUtils.hasRootWorkspace()
    ? sfdxProjectConfig()
    : sfdxGlobalConfig();
  return pathToSFDXConfig;
}

```